js.go

```diff
@@ -192,6 +195,20 @@ func DirectOnly() JSOpt {
 	})
 }
 
+type MaxRetries uint
+
+func (v MaxRetries) configureJSContext(opts *jsOpts) error {
+	opts.maxTries = int(v)
+	return nil
+}
+
+type RetryBackoff time.Duration
+
+func (v RetryBackoff) configureJSContext(opts *jsOpts) error {
```

Comment:
Maybe apply these to some of the tests to simplify? Like:

https://github.com/nats-io/nats.go/blob/master/test/js_test.go#L3205-L3218

and

https://github.com/nats-io/nats.go/blob/master/test/js_test.go#L3843-L3855